src/ontology/cl-edit.owl

```diff
@@ -32821,6 +32823,19 @@ SubClassOf(obo:CL_4052019 obo:CL_0000066)
 SubClassOf(obo:CL_4052019 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0003889))
 SubClassOf(obo:CL_4052019 ObjectSomeValuesFrom(obo:CL_4030045 obo:GO_0005929))
 
+# Class: obo:CL_4052020 (lung resident granzyme K-associated CD8 T cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "doi:/10.1101/2022.06.17.496207") obo:IAO_0000115 obo:CL_4052020 "A granzyme K-associated CD8 T cell that resides in the lung, characterized by the expression of granzyme K (GZMK), and tissue residency markers CD103 and CD49a. This cell exhibits cytotoxic potential through its expression of multiple granzymes (GZMA, GZMB, GZMH, GZMM) in addition to GZMK.")
```

Comment:
The genus is not a cell type in CL. I propose two options:
1) "A  CD8 T cell that resides in the lung, characterized by the expression of granzyme K (GZMK), ..." Since granzyme K is mentioned in the differentia, all the information is still in the definition.
2) Add granzyme K-associated CD8 T cell as a new term.